templates/github/.github/ISSUE_TEMPLATE/task.md.j2

```diff
@@ -0,0 +1,10 @@
+---
+name: 🗒️ Task
+about: Documentation changes, CI/CD changes, work items such as "investigate X"
+title: ''
+labels: Task, Triage-Needed
```

Comment:
A task does not need to be flagged as `Triage-Needed`.

Reply:
Per the old process, yes.

But I don't think there's a great reason to exclude stories or tasks from the triage process personally.  They all ought to have some level of review.

Reply:
on the previous process, 
the default was an issue and everything went to triage, and users rarely changed that.